src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,119 @@
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from '../../utils/SearchLink';
+import { getSearchWith } from '../../utils/searchHelper';
+
+export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const sex = searchParams.get('sex');
+  const query = searchParams.get('query') || '';
+  const centuries = ['16', '17', '18', '19', '20'];
+  const centuriesParams = searchParams.getAll('centuries');
+
+  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchParams(getSearchWith(
+      searchParams,
+      { query: event.target.value || null },
+    ));
+  };
+
+  return (
+    <nav className="panel">
+      <p className="panel-heading">Filters</p>
+
+      <p className="panel-tabs" data-cy="SexFilter">
+        <SearchLink
+          params={{ sex: null }}
+          className={classNames({ 'is-active': !sex })}
+        >
+          All
+        </SearchLink>
+        <SearchLink
+          params={{ sex: 'm' }}
+          className={classNames({ 'is-active': sex === 'm' })}
+        >
+          Male
+        </SearchLink>
+        <SearchLink
+          params={{ sex: 'f' }}
+          className={classNames({ 'is-active': sex === 'f' })}
+        >
+          Female
+        </SearchLink>
+      </p>
+
+      <div className="panel-block">
+        <p className="control has-icons-left">
+          <input
+            data-cy="NameFilter"
+            type="search"
+            className="input"
+            placeholder="Search"
+            value={query}
+            onChange={handleQueryChange}
+          />
+
+          <span className="icon is-left">
+            <i className="fas fa-search" aria-hidden="true" />
+          </span>
+        </p>
+      </div>
+
+      <div className="panel-block">
+        <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
+          <div className="level-left">
+            {centuries.map(century => (
+              <SearchLink
+                params={
+                  {
+                    centuries: centuriesParams.includes(century)
+                      ? centuriesParams.filter(c => c !== century)
+                      : [...centuriesParams, century],
+                  }
+                }
+                key={century}
+                data-cy="century"
+                className={classNames(
+                  'button',
+                  'mr-1',
+                  { 'is-info': centuriesParams.includes(century) },
+                )}
+              >
+                {century}
+              </SearchLink>
+            ))}
+          </div>
+
+          <div className="level-right ml-4">
+            <SearchLink
+              data-cy="centuryALL"
+              className={
+                classNames(
+                  'button',
+                  'is-success',
+                  { 'is-outlined': centuriesParams.length !== 0 },
```

Comment:
pls [see falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)
```suggestion
                  { 'is-outlined': !!centuriesParams.length },
```